src/main/java/net/syscon/elite/repository/jpa/model/ScheduledEvent.java

```diff
@@ -0,0 +1,62 @@
+package net.syscon.elite.repository.jpa.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.JoinColumnOrFormula;
+import org.hibernate.annotations.JoinColumnsOrFormulas;
+import org.hibernate.annotations.JoinFormula;
+import org.hibernate.annotations.NotFound;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import static net.syscon.elite.repository.jpa.model.ScheduledEventType.EVENT_TYPE;
+import static org.hibernate.annotations.NotFoundAction.IGNORE;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "OFFENDER_IND_SCHEDULES")
+public class ScheduledEvent {
```

Comment:
Tempted to remove **ScheduledEvent** and **ScheduledEventType** because these files aren't actually being used. Initially I went down this route but abandoned it for an simpler (subquery) approach given my requirements. It could be useful and it adds to the JPA entity collection? Happy to remove if not.